application/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main(string[] args)
 
 		    using (var amz = new AmazonSynchronizer(new Configuration()))
 		    {
-                amz.WithInterval(TimeSpan.FromMinutes(1))
+                amz.WithInterval(TimeSpan.FromMinutes(5))
```

Comment:
Is this right?  Shouldn't this be 1 second or am I misunderstanding the purpose of this interval?  Also, the default is set to 5 hours, is that right?